### ISSUE TITLE
Multichannel Timeseries: don't rely on custom downloads

### ISSUE THREAD

**Issue by maximlt** [holoviz-topics/examples#448]
For reasons unrelated to examples.holoviz.org (see https://github.com/holoviz-topics/examples/pull/418#discussion_r1847559977), @droumis has implemented inlined downloads (via pooch) on top of anaconda-project `downloads` for the *Multichannel Timeseries* example. I would prefer not to include that sort of code, we cannot guarantee users they will be able to run the notebooks in any other way than via `anaconda-project`. It's not that I have a passion for this tool, it's just that this whole project is too big to offer custom runtime for each and every example.

**Comment by droumis** [comment]
> Multichannel Timeseries: don't rely on custom downloads #448

I don't think this title is a fair assessment of the situation and has led to an internal misunderstanding already. The example does not "rely" on custom downloads. Before this issue was created, it was using anaconda-project for all the data downloads. The only extra layer is that in the notebook, there is a check if the data is downloaded to the location that anaconda-project should download the data to, and if the data is not present, there a is custom download solution to do so.

**Comment by droumis** [comment]
My solution ensures that a user does not require anaconda-project, but anaconda-project is still the encouraged method of running the project, per the website's instructions.
